src/stubs.ts

```diff
@@ -190,8 +169,9 @@ export function stubComponents(
       stubs['transition-group']
     ) {
       return [
-        createTransitionStub({
-          name: 'transition-group-stub'
+        createStub({
+          name: 'transition-group',
+          renderStubDefaultSlot: true
```

Comment:
I guess this will break some shallow snapshots, but probably fine.

Reply:
The `-stub` is added inside of `createStub`

https://github.com/vuejs/vue-test-utils-next/blob/cee2c9ccceef04b256968dd0010d924e21b4c000/src/stubs.ts#L70-L76